nussl/audio_signal.py

```diff
@@ -108,16 +109,34 @@ def __str__(self):
     ##################################################
     #                   Plotting
     ##################################################
-
-    # def plot_time_domain(self):
-    #     """
-    #     Not implemented yet -- will raise and exception
-    #     Returns:
     #
-    #     """
-    #     signal = self.audio_data
-    #     for channel in num_channels
 
+    def plot_time_domain(self, x_label='time', save=False, name=None, output_path=None):
```

Comment:
Some questions: How do I plot only one channel? Can I make a stereo graph like librosa? What if I want to normalize the waveform (to [-1, 1])? Is the image size always the matplotlib default (i.e., you should change it to better aspect ratio if it needs it.

Look at some of the other plotting functions in `nussl`: Repet, RepetSim, AudioSignal.plot_spectrogram, Duet. Look at the logic, the options they have, how the manage user input, etc, and steal liberally.